packages/qwik-city/middleware/bun/index.ts

```diff
@@ -75,7 +75,11 @@ export function createQwikCity(opts: QwikCityBunOptions) {
         mode: 'server',
         locale: undefined,
         url,
-        env: Bun.env,
+        env: {
+          get(key) {
```

Comment:
question

Shouldn't you `...Bun.env` before?
Shouldn't you use `Object.defineProperty()` so that `writable` can be set to `false` for `get`?

Reply:
Don't know about setting writable to false, but this is exactly how it is implemented for node too: https://github.com/BuilderIO/qwik/blob/main/packages%2Fqwik-city%2Fmiddleware%2Fnode%2Fhttp.ts#L85-L89

I don't think we need to add `...Bun.env` either, `env.get` is the only documented way to read environment variables.